packages/graphql/tests/schema/subscriptions.test.ts

```diff
@@ -3974,6 +3974,10 @@ describe("Subscriptions", () => {
               owner: AgreementOwnerConnectFieldInput
             }
 
+            type AgreementConnectedRelationships {
```

Comment:
It looks like `@subscription(operations: [])` isn't getting rid of everything `@exclude(operations: [SUBSCRIBE])` was previously. Is this something we should look at before releasing 4.0?

Reply:
Looking at this, I think it might even be correct now. Previously, it seems like "both sides" were removed -  but that shouldn't really be happening. Now, the subscription field is only removed for the type that you put the directive on. By the looks of it anyway!